lib/cog_api/http/relay_groups.ex

```diff
@@ -38,30 +38,19 @@ defmodule CogApi.HTTP.RelayGroups do
     |> ApiResponse.format_delete("The relay group could not be deleted")
   end
 
-  def update_memberships(action, %{name: name}, %{relay: relay_name}, %Endpoint{}=endpoint) do
-    case get_group_relay(name, relay_name, endpoint) do
-      {:ok, relay, relay_group} ->
-        update_membership(relay_group.id, relay.id, action, endpoint)
-      error ->
-        error
+  def update_memberships(action, %{name: name}, %{relays: relay_names}, endpoint) when is_list(relay_names) do
```

Comment:
Yeah, I agree about the maps. I'm taking this as an iterative refactor. The function is called publicly with either a map or an id. Both update_memberships and update_assignments use the same pattern. I'm thinking that we can clean them up and consolidate the code a bit, but like I said, I was saving that for a later refactor.